src/main/scala/beam/router/r5/R5Wrapper.scala

```diff
@@ -449,6 +449,7 @@ class R5Wrapper(workerParams: R5Parameters, travelTime: TravelTime, travelTimeNo
       }
       streetRouter.profileRequest = profileRequest
       streetRouter.streetMode = toR5StreetMode(vehicle.mode)
+      val legMode = vehicle.mode.r5Mode.get.left.getOrElse(LegMode.valueOf(""))
```

Comment:
Should we map over the `r5Mode` if you are trying to handle `get`'s better?

```suggestion
      val legMode = vehicle.mode.r5Mode.flatmap(_.left.toOption).getOrElse(LegMode.valueOf(""))
```

Reply:
applied this fix